.github/workflows/ci.yml

```diff
@@ -32,7 +32,7 @@ jobs:
     name: "cargo clippy"
     strategy:
       matrix:
-        os: [ubuntu-latest, windows-latest]
+        os: [ ubuntu-latest, windows-latest ]
```

Comment:
What is this madness formatting? :D

Reply:
pycharm

Reply:
https://github.com/astral-sh/uv/pull/2406

Reply:
(I'm not fixing this PR until the underlying bug it fixed)